SPECS/tdnf/tdnf.spec

```
@@ -16,7 +20,9 @@ Source4:        tdnfrepogpgcheck.conf
 Patch0:         tdnf-mandatory-space-list-output.patch
 Patch1:         tdnf-default-mariner-release.patch
 Patch2:         tdnf-enable-plugins-by-default.patch
-Patch3:         tdnf-add-download-command.patch
+Patch3:         tdnf-printf-fix.patch
+Patch4:         tdnf-sqlite-library.patch
+Patch5:         tdnf-GetRepoMD-fix.patch
```

Could you add a link to [the PR fixing this issue upstream](https://github.com/vmware/tdnf/pull/432), so we know what to track and can remove it whenever we update to a version of TDNF, which already contains the upstream fix?
```suggestion
// Patch to be removed once we upgrade to a version of TDNF containing the upstream fix:
// https://github.com/vmware/tdnf/pull/432
Patch5:         tdnf-GetRepoMD-fix.patch
```